media-video/gaupol/gaupol-1.2.ebuild

```diff
@@ -35,11 +35,23 @@ DEPEND="${RDEPEND}
 
 DOCS=( AUTHORS.md NEWS.md TODO.md README.md README.aeidon.md )
 
+src_prepare() {
```

Comment:
I'm pretty sure this will fail due to not calling `eapply_user`, since you implicitly stripped the distutils-r1 phase.